.circleci/config.yml

```diff
@@ -1231,14 +1231,14 @@ workflows:
                 at: /
           workflow-name: "GCP-Daily-Services-Recovery-4N-1C"
 
-  AWS-Daily-Services-Comp-Basic-Performance-4N-4C:
+  AWS-Daily-Services-Comp-Restart-Performance-Random-4N-4C:
```

Comment:
This might be a typo. we should not enable AWS tests

Reply:
Yeah, fixed.